### PR TITLE
Feature/is logged middleware

### PR DESCRIPTION
**Pull request by felmez**

### CODE REVIEW

File: src/middlewares/isLogged.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+  return res
+    .clearCookie('token')
+    .status(400)
+    .json({ error: 'token not found try to login' });
+};
+
+module.exports = {
+  isLogged,
+  SECRET_KEY,
```
Comment (halitbatur) [inline src/middlewares/isLogged.js:33]: why is the SECRET_KEY getting exported form here?

Comment (felmez) [reply src/middlewares/isLogged.js]: i had this idea of not just using the same secret key for all operations in the app but no need for that here, fixed

**Review by halitbatur — APPROVED** [review]
@felmez LGTM

File: src/middlewares/isLogged.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    try {
+      const loggedUser = jwt.verify(token, SECRET_KEY);
+      if (loggedUser) {
+        req.user = loggedUser;
+        next();
+      }
+    } catch (error) {
+      return res
+        .clearCookie('token')
+        .status(400)
```
Comment (halitbatur) [inline src/middlewares/isLogged.js:20]: fix status code as you did with the other middleware and then its ready to be merged

**Review by halitbatur — APPROVED** [review]

**Pull request merged**